src/langs/en.ts

```diff
@@ -82,10 +82,17 @@ export default {
     NO_DATA_FOR_GENERAL_CHARTS: 'No data found for the charts.',
     ATTEMPTS_PROGRESS_NUMBER_OF_ATTEMPTS: 'Number of attempts',
     CREATE_VIEW_NUMBER_OF_ATTEMPTS: 'Number of attempts',
-    MULTIPLE_CHOICE_NOT_CORRECT: 'The answer is incorrect or not completely correct.',
+    MULTIPLE_CHOICE_NOT_CORRECT:
+      'The answer is incorrect or not completely correct.',
```

Comment:
```suggestion
      'The answer is incomplete and/or contains incorrect choices.',
```
We can discuss the formulation I guess.

Reply:
I like it :)